docs/development/build-instructions-gn.md

```diff
@@ -12,6 +12,8 @@ Check the build prerequisites for your platform before proceeding
 * [Linux](build-instructions-linux.md#prerequisites)
 * [Windows](build-instructions-windows.md#prerequisites)
 
+Building Electron can require >200GB disk space between source code and temporary files/caches (depending on file system), and >6 hours compile time (depending on hardware).
```

Comment:
> Building Electron can require >200GB disk space

I'm not sure this is accurate, I quickly checked my local machine and between the `.git-cache` and the `src` folder (which I think covers everything) it's only consuming around ~50GB.  What platform did you pull your numbers from, my 50GB number seems consistent across all 3 platforms.

Reply:
So far I've built on 2 platforms: an APFS volume on an arm64 Mac, and an exFAT volume on Win10. The disk usage obvs varies because of different platform-specific deps but the sheer number of small files involved means that on large block size file systems your disk space is screwed 🥲 . I learned this the hard way building on an exFAT external ssd I happened to have lying around, see pic, yes that's well over 200GB used! Building on an external drive also allowed me to separately observe the amount of temp space needed throughout the process on my main NTFS boot drive which peaked at an additional 35GB or so. On Mac APFS it's about 100GB for the source + a completed build, and several more GBs of temp/cache/global dep installs on top of that.
![Untitled](https://user-images.githubusercontent.com/1031177/182454711-7d07f3fe-a92c-41fb-976c-06f968a82094.png)
<img width="377" alt="Screen Shot 2022-08-02 at 3 09 01 PM" src="https://user-images.githubusercontent.com/1031177/182454737-ec9f32cf-3866-4b33-8b76-e279bd53eb6b.png">